hibike/lib/hibike/hibike_message.cpp

```diff
@@ -81,90 +81,69 @@ int read_message(message_t *msg) {
 
 }
 
-int send_subscription_response(hibike_uid_t* uid, uint16_t delay) {
+int send_subscription_response(uint16_t params, uint16_t delay, hibike_uid_t* uid) {
   message_t msg;
   msg.messageID = SUBSCRIPTION_RESPONSE;
   msg.payload_length = 0;
 
-  int status = 0;
+  int status = append_payload(&msg, (uint8_t*) &params, sizeof(params));
+  
+  status += append_payload(&msg, (uint8_t*) &delay, sizeof(delay));
+  
   status += append_payload(&msg, (uint8_t*) &uid->device_type, UID_DEVICE_BYTES);
   status += append_payload(&msg, (uint8_t*) &uid->year, UID_YEAR_BYTES);
   status += append_payload(&msg, (uint8_t*) &uid->id, UID_ID_BYTES);
 
-  status += append_payload(&msg, (uint8_t*) &delay, sizeof(delay));
 
   if (status != 0) {
     return -1;
   }
   return send_message(&msg); 
 }
 
-
-int send_data_update(uint8_t* data, uint8_t payload_length) {
+//deprecated but not really
```

Comment:
Uh... what is this comment supposed to mean?

If this is not actually deprecated, the comment should be more descriptive. If it is, why is the code still here?

If this not being deleted yet has to do with the order you're submitting PRs, then let me know (odd intermediate states is a normal occurrence, and usually isn't a problem)